compiler/passes/scopeResolve.cpp

```diff
@@ -1548,6 +1548,23 @@ static void resolveUnresolvedSymExpr(UnresolvedSymExpr* unresolvedSymExpr,
 
           unresolvedSymExpr->replace(prim_capture_fn);
           prim_capture_fn->insertAtTail(unresolvedSymExpr);
+          //
+          // If we detect that this function reference is within a
+          // c_fnPtrTo() call then we only need a C pointer to the
+          // function, not a full Chapel first-class function (which
+          // can capture variables).  We'll mark this so that function
+          // resolution can distinguish by adding a second sentinel
+          // argument to the PRIM_CAPTURE_FN call ('true',
+          // arbitrarily).  We could alternatively add another
+          // primitive to distinguish the C case, but for most
+          // intents, it behaves identically to PRIM_CAPTURE_FN, so
+          // this seemed like a waste of a primitive...
+          //
+          // TODO: Can we avoid strcmp or ensure it's "our" fn?
+          //
+          if (call && call->isNamed("c_fnPtrTo")) {
```

Comment:
Oh yeah... duh.  I think I'm always asking Elliot to do this same change...  For some reason I didn't think it would work here, but now I can't see why it wouldn't.